### PR TITLE
How to do local development nowadays

### PR DESCRIPTION
**Pull request by marjuszkiewicz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,15 @@ products and subscriptions, etc.
 
 ## Getting started
 
+You can do local development using our `dev` or `pre` environment.
+
+Since browsers started to blocking Third-Party Cookies, your top domain from local machine needs to
```
Comment (zamzterz) [inline README.md:20]: Typo: "started to blocking" -> "started to block"

File: README.md [hunk 2]
```diff
@@ -15,6 +15,15 @@ products and subscriptions, etc.
...
+match yours session service top domain. Otherwise session service cookie will be Third-Party Cookie,
```
Comment (zamzterz) [inline README.md:21]: Typo: "yours" -> "your"

File: README.md [hunk 3]
```diff
@@ -15,6 +15,15 @@ products and subscriptions, etc.
...
+and will not be send with XHR request.
```
Comment (zamzterz) [inline README.md:22]: Typo: "send" -> "sent"

File: README.md [hunk 4]
```diff
@@ -15,6 +15,15 @@ products and subscriptions, etc.
...
+
+For example if your `pre` domain is pre.sdk-example.com, and it uses id.pre.sdk-example.com session
+service domain, your local domain should be local.sdk-example.com.
```
Comment (zamzterz) [inline README.md:25]: This needs to be `local.pre.sdk-example.com` (subdomain of `pre.sdk-example.com`).

**Review by zamzterz — CHANGES_REQUESTED** [review]
Nice!

The example local domain should be a subdomain of the example session-service domain.

**Review by zamzterz — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (erikfried) [reply README.md:25]: Might also be worth mentioning that you require local https to get this working (if I am not mistaken).

Comment (marjuszkiewicz) [reply README.md:25]: We have verified that `local.sdk-example.com` is enough. Works without https for me

**Pull request merged**